src/template.js

```diff
@@ -0,0 +1,64 @@
+var globby = require('globby');
+var path  = require('path');
+var utils = require('./utils');
```

Comment:
Did we decide to stick with CommonJS modules for the sake of unit testing?

Also, how about `const` for these?